kafka/stackable/patches/3.9.0/0002-Use-stackable-maven-mirror.patch

```diff
@@ -34,5 +34,5 @@ index 92082fe7cf..3b56a2ad98 100644
  
    dependencyUpdates {
 -- 
-2.47.1
+2.40.1
```

Comment:
question: Is this correct? Going to an older versions sees counter-intuitive.

Reply:
TIL this is the git version. It will always depend on who did the patch :shrug: 

```
❯ git --version                                                   
git version 2.40.1
```